Sources/BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -285,6 +297,11 @@ - (void)updateRequiredFields {
     }
 }
 
+- (void) tappedPrivacyPolicy: (UITapGestureRecognizer *) gestureRecognizer {
+    NSURL *privacyPolicy = [NSURL URLWithString:@"https://www.paypal.com/us/legalhub/home?locale.x=en_US"];
+    [[UIApplication sharedApplication] openURL:privacyPolicy options:@{} completionHandler:nil];
```

Comment:
We can also consider opening this an an `ASWebAuthenticationSession` if we want to keep this in context

Reply:
I think ASWeb would be better for conversion. But we should keep it consistent across SDKs. Android would have to use browser switch a well.